src/rpcserver.cpp

```diff
@@ -357,23 +357,25 @@ static const CRPCCommand vRPCCommands[] =
 
     /* Omni Core data retrieval calls */
     /* CATEGORY                              NAME                               ACTOR (FUNCTION)                  OKSAFEMODE  THREADSAFE  REQWALLET */
-    { "omni layer (data retrieval)",         "omni_getinfo",                    &omni_getinfo,                    true,       true,       false },
```

Comment:
Uh, oh!

Looks like this is the old version (probably introduced when resolving conflicts).